src/components/adaptive-cards/hooks/useAction.js

```diff
@@ -0,0 +1,15 @@
+import useActionOpenUrl from './useActionOpenUrl';
+import useActionSubmit from './useActionSubmit';
+import useActionToggleVisibility from './useActionToggleVisibility';
+
+/**
```

Comment:
Custom hook that receives an adaptive card action and returns html attributes for the target element